src/SumoCoders/FrameworkCoreBundle/Helper/InstallerHelper.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+namespace SumoCoders\FrameworkCoreBundle\Helper;
+
+use Composer\IO\IOInterface;
+use Symfony\Component\Yaml\Dumper;
+use Symfony\Component\Yaml\Exception\ParseException;
+use Symfony\Component\Yaml\Parser;
+use Symfony\Component\Yaml\Yaml;
+
+class InstallerHelper
```

Comment:
Why InstallerHelper and not just Installer? I don't think the Helper part of the name adds any value to the class name.